charts/cluster-scanner/values.yaml

```diff
@@ -0,0 +1,124 @@
+#basic Default values for cluster-scanner.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  sysdigHost: ""
+  sysdigAccessKey: ""
+  clusterName: ""
+  sysdigSkipVerifyCertificate: false
+
+  scannerMode: "local" # or "multi"
+  loggingLevel: "INFO"
+
+eveEnabled: false
+eveIntegrationEnabled: false
+rootNamespace: "kube-system"
+
+multicluster:
+  # kubeconfigSecretName: ""
+
+localcluster:
+  # restrict access to specific docker secrets to obtain pull credentials. Optional.
+  # Default behaviour is listing all secrets
+  # rbac:
+  #   allowedPullSecrets:
+  #     namespaceOneName:
+  #       - myDockerSecretOne
+  #       - myDockerSecretTwo
+  #     anotherNamespace:
+  #       - myOtherDockerSecretOne
+  #       - myOtherDockerSecretTwo
+
+replicaCount: 2
+
+# Params to manage leader election of rsi component.
+# Leader election is implemented leveraging the native capabilities of Kubernetes
+# see: https://kubernetes.io/blog/2016/01/simple-leader-election-with-kubernetes/
+# leaderElectionLeaseNameOverride: ""
+# leaderElectionLeaseNamespaceOverride: ""
+
+rsi:
```

Comment:
Can we consider to use the full name here? I think it become more understandable for a user then using the acronym.. (same for `ise` a few line below)